css/modules/styles.css

```diff
@@ -4436,8 +4436,8 @@ span#usertab_text {
 }
 
 #virtualclassCont.congrea #fullScreenButton .fullScreen,
-#virtualclassCont.congrea #fullScreenExitButton [data-title="Exit Full Screen"] {
-  padding: 0.6875rem 1.125rem 0.6875rem 1.125rem;
+#virtualclassCont.congrea #fullScreenExitButton span {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector "#virtualclassCont.congrea #fullScreenExitButton span" to come before selector "#virtualclassCont.congrea.student #resultLayoutBody #optnNonVotd span #virtualclassCont.congrea.student #layoutPollBody span" (no-descending-specificity)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)